src/transport/grpc/mod.rs

```diff
@@ -478,7 +477,7 @@ mod test {
     enum ServerReceivedRequest {
         Attach(Node),
         Send(AstarteMessage),
-        Detach(Node),
+        Detach(astarte_message_hub_proto::pbjson_types::Empty),
```

Comment:
I would just import the Empty struct since it's used in lot's of places.

Reply:
Yep, could be a good idea